varcode/frameshift_coding_effect.py

```diff
@@ -118,8 +118,48 @@ def _frameshift(
         transcript=transcript,
         aa_mutation_start_offset=mutation_start_position,
         aa_ref=aa_ref,
-        shifted_sequence=mutant_protein_suffix)
+        shifted_sequence=str(mutant_protein_suffix))
 
+def cdna_codon_sequence_after_insertion_frameshift(
+        sequence_from_start_codon,
+        cds_offset_before_insertion,
+        inserted_nucleotides):
+    """
+    Returns index of mutated codon and nucleotide sequence starting at the first
+    mutated codon.
+    """
+    # special logic for insertions
+    coding_sequence_after_insertion = \
+        sequence_from_start_codon[cds_offset_before_insertion + 1:]
+    if cds_offset_before_insertion % 3 == 2:
```

Comment:
Maybe it'll be helpful to add more unit-y tests, e.g. something like this for each branch of logic, vs. relying on the more end-to-end tests?

```
cdna_codon_sequence_after_insertion_frameshift(
    sequence_from_start_codon="ACTGAA",
    cds_offset_before_insertion=0,
    inserted_nucleotides="T")
```